app/constants/languageMenu.js

```diff
@@ -11,6 +11,7 @@ export default {
   he: 'עברית',
   hi: 'हिन्दी',
   hr: 'Hrvatski',
+  hu: 'magyar nyelv',
```

Comment:
```suggestion
  hu: 'Magyar',
```

Reply:
Following example from Wikipedia’s language selector (https://www.wikipedia.org/) and others (e.g. https://confluence.atlassian.com/proforma/internationalization-1087521479.html)